requirements/run.in

```diff
@@ -13,5 +13,7 @@ watchdog==2.1.9
 pyjwt==2.4.0
 pymongo==4.1.0
 pydantic==1.9.0
+dnspython==2.2.1
+email-validator==1.3.0
```

Comment:
@Alopalao, well done pinning the dependencies here. Regarding their transitive dependencies below we also need to make sure whatever is pinned is compatible with the other NApps runtime dependencies, I'll open some threads discussing them with you.

Reply:
Ok.